### PR TITLE
changes for helm v3

### PR DESCRIPTION
**Pull request by galal-hussein**
https://github.com/rancher/klipper-helm/pull/3

### CODE REVIEW

File: pkg/helm/controller.go [hunk 1]
```diff
@@ -297,15 +297,15 @@ func serviceAccount(chart *helmv1.HelmChart) *core.ServiceAccount {
 func args(chart *helmv1.HelmChart) []string {
 	if chart.DeletionTimestamp != nil {
 		return []string{
-			"delete",
-			"--purge", chart.Name,
+			"uninstall",
```
Comment (ibuildthecloud) [inline pkg/helm/controller.go]: Does this mean a helm 2 chart won't delete now?

Comment (galal-hussein) [reply pkg/helm/controller.go]: no that was an old update that meant to only use v3, pushing the new update now

Comment (galal-hussein) [reply pkg/helm/controller.go]: @ibuildthecloud The new update will pass only delete/install and the script will handle the installation or deletion for helm v2 and v3

**Comment by ibuildthecloud** [comment]
Can we add a field to spec, `helmVersion` that can be 2 or 3 and default 3.  Then a user can force the controller to still use helm 2 for whatever reason.

**Pull request merged**